docs/.vuepress/config.js

```diff
@@ -34,7 +34,7 @@ module.exports = {
     displayAllHeaders: true,
     lastUpdated: 'Last Updated',
     nav: [
-      { text: 'Talk to Us', link: 'https://discord.gg/74GUQDE' },
+      { text: 'Talk', link: 'https://discord.gg/74GUQDE' },
```

Comment:
I don't like that either.

What about `Reach Out` and `Contact (Us)`; or `Discuss`, `Communicate` and `Connect`?